Assets/Thirdweb/Examples/Scripts/Utils/Debugger.cs

```diff
@@ -0,0 +1,31 @@
+using UnityEngine;
+using TMPro;
+
+public class Debugger : MonoBehaviour
+{
+    public GameObject debuggerPanel;
+    public TMP_Text titleText;
+    public TMP_Text descriptionText;
+
+    public static Debugger Instance;
+
+    private void Awake()
+    {
+        if (Instance == null)
+            Instance = this;
+        else
+            Destroy(this.gameObject);
+    }
+
+    private void Start()
+    {
+        debuggerPanel.SetActive(false);
+    }
+
+    public void Log(string title, string description)
+    {
+        debuggerPanel.SetActive(true);
```

Comment:
Haven't tested this yet, but is this debugger popup dismissable?

Reply:
Yes it is